### DISCUSSION TITLE
Laravel Sail shows default Apache page

### DISCUSSION THREAD

**Discussion by sbaum2s** [laravel/framework#52119] [Q&A]
Hi, having installed it according official documentation in a fresh laravel project and firing it up with "sail up" i dont see the actual Laravel app on localhost. Is this a bug or am i missing something?

**Comment by taylorivanoff** [comment]
I was using WSL2 and docker for my Laravel Sail setup and had the same problem showing the default Apache page. It was because WSL2 had Apache2 installed pre-installed and. I removed it with `sudo apt remove apache2` and it was resolved.

**Comment by sbaum2s** [comment]
The less disruptive way is to shut down apache service temporarily inside WSL2 with `sudo service apache2 stop`

I am looking for a way to add this hint to the official documentation since this is really annoying and costs a lot of time to figure out and makes you less productive which is one of the main selling points of Laravel.
